templates/partials/footer.hbs

```diff
@@ -1,31 +1,72 @@
   </article>
 
   <footer>
-    <a href="/">Add test</a> &middot;
-
-    {{!-- TODO: this is never added to the context, but set on session after test added
-    https://github.com/mtharrison/hapi-context-credentials/blob/master/index.js
-    --}}
-    {{#if authorSlug}}
-      <a href="/browse/{{authorSlug}}">My tests</a> &middot;
-    {{/if}}
-
-    <a href="/browse">Latest</a> &middot;
-
-    <a href="/popular">Popular</a> &middot;
-
-    <a href="/faq">FAQ</a> &middot;
-
-    <a href="/faq#donate">Donate</a> &middot;
-
-    <a href="https://twitter.com/jsperf" rel="nofollow">twitter: @jsperf</a> &middot;
-
-    <a href="https://github.com/jsperf/jsperf.com">source on GitHub</a> &middot;
-
-    <a href="https://benchmarkjs.com/">Benchmark.js</a> &middot;
-
-    by <a href="https://mathiasbynens.be/">@mathias</a> and <a href="/contributors">contributors</a>
-
+    <ul>
+      <li>
+        <h4>Sponsors</h4>
+        <p>Hosting generously provided by <a href="https://m.do.co/c/b426e15331e8" target="_blank">DigitalOcean</a> and <a href="https://zeit.co/" target="_blank">Zeit</a></p>
+
+        <p><a href="/sponsor">Learn how to become a sponsor</a></p>
+      </li>
+
+      <li>
+        <h4>Tests</h4>
+        <ul>
+          <li>
+            <a href="/">Add</a>
+          </li>
+
+            {{!-- TODO: this is never added to the context, but set on session after test added
+            https://github.com/mtharrison/hapi-context-credentials/blob/master/index.js
+            --}}
+            {{#if authorSlug}}
+              <li>
+                <a href="/browse/{{authorSlug}}">My tests</a> &middot;
+              </li>
+            {{/if}}
+
+          <li>
+            <a href="/browse">Latest</a>
+          </li>
+          <li>
+            <a href="/popular">Popular</a>
+          </li>
+        </ul>
+      </li>
+
+      <li>
+        <h4>Info</h4>
+        <ul>
+          <li>
+            <a href="/faq">FAQ</a>
+          </li>
+          <li>
+            <a href="/testimonials">Testimonials</a>
+          </li>
+          <li>
+            <a href="/contributors">Contributors</a>
+          </li>
+          <li>
+            <a href="https://benchmarkjs.com/">Benchmark.js</a>
+          </li>
+        </ul>
+      </li>
+
+      <li>
+        <h4>Contact</h4>
+        <ul>
+          <li>
+            <a href="https://twitter.com/jsperf" target="_blank">Twitter</a>
+          </li>
+          <li>
+            <a href="https://github.com/jsperf/jsperf.com" target="_blank">GitHub</a>
+          </li>
+          <li>
+            <a href="https://webchat.freenode.net/?channels=jsperf" target="_blank">IRC</a>
```

Comment:
Let’s remove the use of `target="_blank"` everywhere. Developers can decide on their own if they choose to open a link in a new window/tab or not. (If you’d rather keep it, [we should add `rel=noopener`](https://mathiasbynens.github.io/rel-noopener/)).